(2) Vox Populi/Database Changes/UnitPromotions/NewPromotions.xml

```diff
@@ -514,6 +523,13 @@
 			<PediaType>PEDIA_NAVAL</PediaType>
 			<PediaEntry>TXT_KEY_PROMOTION_PRIZE_RULES</PediaEntry>
 		</Row>
```

Comment:
Prize Rules should be removed from the game (+ all references to it) if unused.

Unless some modmods somehow use it?

Reply:
I don't know. I'm not really comfortable deleting promotions.